src/Common/src/Interop/Windows/kernel32/Interop.Access.cs

```diff
@@ -0,0 +1,41 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class Kernel32
+    {
+		/// <summary>
+        /// <a href="https://docs.microsoft.com/en-us/windows-hardware/drivers/kernel/buffer-descriptions-for-i-o-control-codes">RequiredAccess</a>.
+		/// Indicates the type of access that a caller must request when opening the file object that represents the device (see IRP_MJ_CREATE).
+        /// </summary>
+		
+        public enum Access
+        {
+			/// <summary>
+            /// The I/O manager sends the IRP for any caller that has a handle to the file object that represents the target device object.
+			/// </summary>
+            FILE_ANY_ACCESS,
```

Comment:
Should call out that this value should be avoided. (https://docs.microsoft.com/en-us/previous-versions/windows/hardware/design/dn614603) Also, consider actually putting in the defined values explicitly (i.e. 0x00, 0x01, etc.)